packages/main/src/index.ts

```diff
@@ -99,3 +99,29 @@ app.whenReady().then(() => {
 ipcMain.on('dialog:show-error', (_, param: { title: string; body: string }): void => {
   dialog.showErrorBox(param.title, param.body);
 });
+
+ipcMain.handle(
+  'dialog:show-dialog',
+  async (_, param: { type: string; title: string; message: string; items: string[] }): Promise<string | undefined> => {
+    const window = BrowserWindow.getAllWindows().find(w => !w.isDestroyed());
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const result = await dialog.showMessageBox(window!, {
```

Comment:
would it work with a `?` instead of `!`

Reply:
no, as `?` expects expression after

Reply:
ok so probably helper method with an error thrown if it has not been found